backend/app/views.py

```diff
@@ -186,21 +188,24 @@ def image_to_music(request):
 
 
 @api_view(['POST'])
-def csv_upload(request):
-    """
-    Handling extracting data from user-inputted CSV file
-    """
-    tempfile = request.FILES.get('value')
-    csvdata = encoders.parse_csv_upload(tempfile)
-    return Response(csvdata)
-
-
-@api_view(['POST'])
-def change_settings(request):
-    """
-    Handling settings changes from frontend
-    """
-    print("type:", request.data.get('type'),
-          "id:", request.data.get('id'),
-          "value:", int(request.data.get('value')))
-    return Response(request.data)
+def generate_instrument(request):
```

Comment:
The bulk of this probably shouldn't be here in a view function, but doing this quick and dirty for now. Will refactor next week.